packages/lesswrong/components/posts/PostsPreviewTooltip.tsx

```diff
@@ -58,6 +58,9 @@ const styles = (theme: ThemeType): JssStyles => ({
       cursor: "pointer"
     }
   },
+  postPreview: {
```

Comment:
I had given PostsPreviewTooltip a maximum height to prevent some weirdly long post-previews from having erratic behavior when you hovered over post-items. (Posts with either large header-text, or tables, or lots of whitespace, both ended up being cumbersome)

BUT this had a bad side effect for post comment previews (i.e. postpreviews that link to a specific comment on that post). Normally you can click to expand them, but the expanded version was truncated.

This came up in this PR because it was dealing with review-comments, and how they appear in LatestReviews.tsx. 

I moved the maxHeight to only affect post-previews, not comment-previews.